source/configuration_manual/mail_crypt_plugin.rst

```diff
@@ -350,25 +334,34 @@ similarly to the fs-compress wrapper. It can be used to encrypt e.g.:
 * FTS index objects (fts_dovecot_fs)
 * External mail attachments (mail_attachment_fs)
 
-fs-crypt comes in two flavors, mail-crypt and crypt. mail-crypt is intended to
-be used with user context, while crypt can be used elsewhere. Note that
-fs-[mail-]crypt and the fs-compress wrapper can be also combined. Please make
-sure that compression is always applied before encryption. See
+fs-crypt comes in two flavors, *mail-crypt* and *crypt*. *mail-crypt* is
+intended to be used with user context, while *crypt* can be used elsewhere.
```

Comment:
s/user context/mail storage context/ maybe? or maybe "mail user". Of course, none of these are very clear to non-Dovecot-coder what it means exactly.. But I'm not sure if it can be. "works in some places" :)